### PR TITLE
baseline: Fix barfing on NaNs when there's no data.

### LINKED ISSUE #131: Baseline plotting bug
Came up while testing v0.19-rc0-52 today.

image

### PR DESCRIPTION
**Pull request by gsmcmullin**
Fixes #131

### CODE REVIEW

**Comment by henryhallam** [comment]
`if not np.all(np.isnan(neds)):`

File: piksi_tools/console/baseline_view.py [hunk 1]
```diff
@@ -219,13 +219,14 @@ def baseline_callback(self, sbp_msg):
...
-    self.plot_data.set_data('d_fixed', neds_fixed[:][2])
-
-    self.plot_data.set_data('n_float', neds_float[:][0])
-    self.plot_data.set_data('e_float', neds_float[:][1])
-    self.plot_data.set_data('d_float', neds_float[:][2])
+    if not all(map(any, np.isnan(neds_fixed))):
+      self.plot_data.set_data('n_fixed', neds_fixed.T[0])
+      self.plot_data.set_data('e_fixed', neds_fixed.T[1])
+      self.plot_data.set_data('d_fixed', neds_fixed.T[2])
+    if not all(map(any, np.isnan(neds_float))):
```
Comment (denniszollo) [inline piksi_tools/console/baseline_view.py]: I agree that this is a little confusing and Henry's check is more clear

Comment (gsmcmullin) [inline piksi_tools/console/baseline_view.py]: Henry's check doesn't work for me.  It complains about truth value of an array and still barfs.

**Pull request merged**